app/services/recurring_contribution/subscriptions/processor.rb

```diff
@@ -54,10 +54,17 @@ def paid_with_credit_card?
 
   def assign_attributes(with_save, attributes)
     subscription.assign_attributes(attributes)
+    update_user_cpf
 
     subscription.save if with_save && !has_errors?
   end
 
+  def update_user_cpf
+    return register_error(:donator_cpf, :blank) if subscription.donator_cpf.blank?
```

Comment:
I also thought this, @fladson. But I don't know if it fits well because this verification is exclusive of the `create` action. I mean, I added this attribute exclusively for be used on the view where the user creates new subscriptions, only on this moment he will send the  `cpf` value. So if I add a validation, on the model, operations like `update` - where this value wont be sent - will raise errors. What do you think about?